### PR TITLE
Strengthify is now in the Nextcloud org

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
     "regenerator-runtime": "^0.13.9",
     "select2": "3.5.1",
     "snap.js": "^2.0.9",
-    "strengthify": "git+https://github.com/MorrisJobke/strengthify.git#0.5.9",
+    "strengthify": "github:nextcloud/strengthify#0.5.9",
```
Comment (nickvergessen) [inline package.json:76]: I did:
```sh
npm uninstall strengthify
npm install strengthify@git+https://github.com/nextcloud/strengthify.git#0.5.9
```
so this seems to be a new format

**Review by skjnldsv — APPROVED** [review]

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**